src/main/java/org/broadinstitute/consent/http/resources/DataRequestVoteResource.java

```diff
@@ -241,7 +246,7 @@ private void validateUserAndVoteId(AuthUser authUser, Integer voteId) throws Not
                 existingVote.getDacUserId());
     }
 
-    private void createDataOwnerElection(Vote vote, DataAccessRequest dar) throws MessagingException, IOException, TemplateException {
+    private void createDataOwnerElection(Vote vote, String referenceId) throws MessagingException, IOException, TemplateException {
```

Comment:
Thank you for updating this. We don't actually create data owner elections, but the code is now compatible. It really should be in a service class, but we'll be removing this in the near future in favor data submitter functionality.